### PR TITLE
[NVA] Update NVA connection cmdlet

### PR DESCRIPTION
**Pull request by rutuja-s**
Update or Change a Network Virtual Appliance Connection resource.

Cmdlet design review accepted PR:  [[NVA] Update NVA connection cmdlet ](https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/1406)

## Mandatory Checklist

- Please choose the target release of Azure PowerShell
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] N/A

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Network/Network/NetworkVirtualAppliance/Connections/UpdateAzureNetworkVirtualConnectionCommand.cs.cs [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+                        WriteObject(this.CreateOrUpdateNetworkVirtualApplianceConnection(this.ResourceGroupName, this.VirtualApplianceName, this.Name, nvaConnectionToModify));
+                    }
+                );
+
+
+            }
+
+
+
+
```
Comment (wyunchi-ms) [inline src/Network/Network/NetworkVirtualAppliance/Connections/UpdateAzureNetworkVirtualConnectionCommand.cs.cs:186]: Please remove the empty lines.

Comment (rutuja-s) [reply src/Network/Network/NetworkVirtualAppliance/Connections/UpdateAzureNetworkVirtualConnectionCommand.cs.cs]: fixed

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**